obs-check-devel-openqa-leap-extra-dependencies

```diff
@@ -2,16 +2,45 @@
 
 osc="osc --apiurl https://api.opensuse.org"
 
-for project in $($osc search --project -s 'devel:openQA:Leap:' | grep -o '^devel:openQA:Leap:.*'); do
-    for package in $($osc list "$project"); do
-        $osc api "/comments/package/$project/$package" \
-            | yq -pxml -oy '.comments.comment |= ([] + .) | .comments.comment[].+content' \
-            | grep -q "Reason for linking:" \
-            || {
-                echo "No reason for $project/$package, consider adding a comment or remove the link (see poo#128087 for details)" >&2
-                problem=1
-            }
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Checks devel:openQA:Leap: OBS project dependencies.
```

Comment:
Please be a bit more descriptive here.
Even reading the source code I'm still not sure what exactly this script does.

Reply:
Yep, me neither

Reply:
If nobody knows what this does, we might want to delete the script.

Reply:
No, the script is actively used within https://gitlab.suse.de/openqa/scripts-ci/